processing/manual_correction.sh

```diff
@@ -26,13 +26,13 @@ yaml_parser() {
   INPUT_YAML=$1
 
   # tr: delete all newlines begining with dash (-), so convert input file into one line without dashes
-  # (unfortunately this also delete all other dashes, so it is neccesary to use 3rd sed)
+  # (this also deletes all other dashes, so it is necessary to use last sed)
   # 1st sed: delete "FILES_LABEL:" string and everything after it
   # 2nd sed: delete "FILES_SEG:" string itself
   # 3rd sed: replace "sub" by "sub-" (tr command deleted all dashes (-))
```

Comment:
@valosekj if you have troubles with the first "-" in the YAML format we chose, why don't you go with the other YAML:
~~~
# FIELD 1
value1
value2
value3
~~~
does that make sense?

Reply:
^ scrap that comment above: i just realized that "#" is considered a comment, not a key, so that wouldn't be proper use of the YAML format (in case in the future we implement a proper yaml parser).

Reply:
Yes, "#" denotes comments. I would let current format which follows [yaml syntax](https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html). Originally, I wanted to use `sed` command for removing `newline` character and first `-`, but i found out that default `sed` on OSX does not support `\n`. So i had to workaround it using `tr` command.

Reply:
>(in case in the future we implement a proper yaml parser).

We could use some of already created yaml parsers (e.g., [here](https://github.com/jasperes/bash-yaml)). On the other hand, if we don't modify structure of yaml file, currently proposed simple dedicated parser could be sufficient.

Mentioned also in https://github.com/sct-pipeline/csa-atrophy/issues/23